test/functorch/test_ops.py

```diff
@@ -1195,6 +1195,8 @@ def vjp_of_vjp(*args_and_cotangents):
                 "linalg.householder_product",
                 {torch.float32: tol(atol=3e-04, rtol=9e-04)},
             ),
+            # Tighten tolerance to check for regressions of issue #114868
+            tol1("linalg.tensorsolve", {torch.float32: tol(atol=1e-4, rtol=1e-5)}),
```

Comment:
Yep, let's remove this one as any change in hardware may make it fail, but yeah, thank you for checking that this indeed solves the regression.

Reply:
If I understand correctly, the iterative version can use a different algorithm than the batched version with very different numerical properties. If that's the case, the point above still stands.

If the two are actually calling exactly the same ops and we can guarantee bitwise equivalence, then let's set atol = rtol = 0.

Reply:
> the iterative version can use a different algorithm

This change removes one place where this was the case yes. From what I remember during my investigation of this issue this now does not use a different algorithm in any case.   
This could change in the future of course but I don't see how you'd implement linalg solver over multiple batches differently than iterating them.

> then let's set atol = rtol = 0.

We can formally do that but it has no effect: There is a minimum tolerance set depending on the data type. So the check wouldn't be for bitwise equivalence but I guess it shows the intention well enough so we could do that.